Frends.AzureBlobStorage.UploadBlob/Frends.AzureBlobStorage.UploadBlob/Definitions/Destination.cs

```diff
@@ -1,114 +1,91 @@
-
-        /// <summary>
-        ///     Azure blob type to upload: Append, Block or Page.
-        /// </summary>
-        [DefaultValue(AzureBlobType.Block)]
-        [DisplayName("Blob Type")]
-        public AzureBlobType BlobType { get; set; }
-
-        /// <summary>
-        ///     Source file can be renamed to this name in azure blob storage.
-        /// </summary>
-        [DefaultValue("")]
-        [DisplayName("Rename source file")]
-        [DisplayFormat(DataFormatString = "Text")]
-        public string RenameTo { get; set; }
-
-        /// <summary>
-        ///     Set desired content-type. If empty, task tries to guess from mime-type.
-        /// </summary>
-        [DefaultValue("")]
-        [DisplayName("Force Content-Type")]
-        [DisplayFormat(DataFormatString = "Text")]
-        public string ContentType { get; set; }
-
-        /// <summary>
-        ///     Set desired content-encoding. Defaults to UTF8 BOM.
-        /// </summary>
-        [DefaultValue("")]
-        [DisplayName("Force Content-Encoding")]
-        [DisplayFormat(DataFormatString = "Text")]
-        public string FileEncoding { get; set; }
-
-        /// <summary>
-        ///     Should upload operation overwrite existing file with same name?
-        /// </summary>
-        [DefaultValue(true)]
-        [DisplayName("Overwrite existing file")]
-        public bool Overwrite { get; set; }
-
-        /// <summary>
-        ///     How many work items to process concurrently.
-        /// </summary>
-        [DefaultValue(64)]
-        [DisplayName("Parallel Operation")]
-        public int ParallelOperations { get; set; }
-    }
-
-    public class UploadInput
-    {
-        [DefaultValue(@"c:\temp\TestFile.xml")]
-        [DisplayName("Source File")]
-        [DisplayFormat(DataFormatString = "Text")]
-        public string SourceFile { get; set; }
-
-        /// <summary>
-        ///     Uses stream to read file content.
-        /// </summary>
-        [DefaultValue(false)]
-        [DisplayName("Stream content only")]
-        public bool ContentsOnly { get; set; }
-
-        /// <summary>
-        ///     Works only when transferring stream content.
-        /// </summary>
-        [DefaultValue(false)]
-        [DisplayName("Gzip compression")]
-        public bool Compress { get; set; }
-    }
-
-    public enum AzureBlobType
-    {
-        Append,
-        Block,
-        Page
-    }
-}
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AzureBlobStorage.UploadBlob
+{
+    /// <summary>
+    /// Options-class for UploadBlob-task.
+    /// </summary>
+    public class Destination
+    {
+        /// <summary>
+        /// Connection string to Azure storage.
+        /// </summary>
+        [DefaultValue("UseDevelopmentStorage=true")]
+        [DisplayName("Connection String")]
+        [DisplayFormat(DataFormatString = "Text")]
+        public string ConnectionString { get; set; }

```

Comment:
Sorry, forgot last time - this connection string should definitely be secret, as it contains access key.

Reply:
Fixed.